modules/workspace.nix

```diff
@@ -170,5 +192,19 @@ in
         priority = 3;
       };
     })
+    (lib.mkIf (cfg.workspace.wallpaperPictureOfTheDay != null) {
+      programs.plasma.startup.desktopScript."set_wallpaper_potd" = {
+        text = ''
+          let allDesktops = desktops();
+          for (var desktopIndex = 0; desktopIndex < allDesktops.length; desktopIndex++) {
+              var desktop = allDesktops[desktopIndex];
```

Comment:
It's currently the year 2024. Plasma's desktop scripts support ES6, even though all their docs are ES5 for some reason
```js
for (const desktop of desktops()) {
    // ...
}
```